editoast/openapi.yaml

```diff
@@ -1722,3 +1752,24 @@ components:
           type: array
           items:
             type: string
+    Geometry:
+      type: object
+      description: Simple definition of a GeoJSon geometry
+      required:
+        - type
+        - coordinates
+      properties:
+        type:
+          type: string
+          description: the geometry type
+          enum:
+            - Point
+            - LineString
+            - Polygon
+            - MultiPoint
+            - MultiLineString
+            - MultiPolygon
+        coordinates:
+          type: array
+          items:
+            type: number
```

Comment:
This is not correct:
- For `Point`, `coordinates` must be `number[]`
- For `MultiPoint` and `LineString`, `coordinates` must be `number[][]`
- For `MultiLineString` and `Polygon`, `coordinates` must be `number[][][]`
- For `MultiPolygon`, `coordinates` must be `number[][][][]`